README.md

```diff
@@ -84,19 +112,28 @@ To include an API user guide, add markdown files in a `user-guide` folder struct
 More information about the content of each markdown file is provided below.
 
 #### overview.md - Technical overview
+
 This overview is aimed at developers, so you should go into more technical detail than the catalogue description.
 Explain what the API does and how it works, e.g. 'it's a FHIR API that lets you retrieve ValueSets resources of SNOMED concepts', it's a SOAP API that triggers a questionnaire to be sent.
 Be sure to highlight any technology and concepts that the user should be familiar with.
 
 #### quickstart.md - How to get started
+
 This should provide a short and clear example of how to use the API. 
 You can assume the reader has some familiarity with the required technology, but highlight anything advanced or unconventional - i.e. gotchas!
 Show sample requests, responses and explain error handling where appropriate
 
 #### how-to - Walk-through guides for common scenarios
+
 These should be examples written in the same style as the quickstart but it's ok for these to be lengthier, more complex examples of usage.
 
-#### concepts - detail on API 
+#### concepts - detail on API
+
 These are descriptions of key concepts specific to the API, e.g. a Document Metadata standard.
-Where relevant, link to information sources (e.g. where concepts reference from some external standard.) 
+Where relevant, link to information sources (e.g. where concepts reference from some external standard.)
+
+## Contributing
+
+The catalogue is in the open and we encourage contributions. So if you come across a spelling mistake or wish to add an NHS API, feel free to send a [pull request]((https://github.com/nwisbeta/api-catalogue/pulls)) or you can open a [issue]([issue](https://github.com/nwisbeta/api-catalogue/issues)) to report it to us.
```

Comment:
The links are slightly malformed, they should be `[text for the link](https://adddress.to/link-to)`

Also it might be better to put this right at the top of the README

Reply:
Agreed. I'll look to implement these suggestions